CODEOWNERS

```diff
@@ -20,7 +21,6 @@ server/chat-plugins/rock-paper-scissors.ts @mia-pi-git
 server/chat-plugins/sample-teams.ts @KrisXV
 server/chat-plugins/scavenger*.ts @xfix @sparkychildcharlie
```

Comment:
should we get rid of sparkychild from this?

Reply:
this is something that can be discussed for another pr, so im merging this as-is for now